.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "vendor/c-ares/c-ares"]
 	path = vendor/c-ares/c-ares
 	url = https://github.com/c-ares/c-ares.git
+[submodule "vendor/tartanllama/expected"]
```

Comment:
Can you update the root README.md to note that this is now pulled by submodule instead of embedded please?

Reply:
Updated README.md